labelImg.py

```diff
@@ -16,6 +18,7 @@
     from PyQt5.QtGui import *
     from PyQt5.QtCore import *
     from PyQt5.QtWidgets import *
+    from PyQt5.QtWebEngineWidgets import *
```

Comment:
what if the users are using pyqt4?

Also, could you update these two files if introducing new dependencies?
https://github.com/tzutalin/labelImg/blob/master/.travis.yml
https://github.com/tzutalin/labelImg/blob/master/README.rst

Reply:
I commented this line, as it's unncessary.

Reply:
I will suggest removing it